help/README.md

```diff
@@ -28,7 +28,9 @@ If you are a developer comfortable working on the command line, you can edit the
 ### The easy way: edit on Github
 If you don't want to set up your own local dev environment, feel free to just edit the help materials directly from Github:
 
-1. ... give instructions here ...
+1. Open whatever file you want.
```

Comment:
Really just including this change so there would be a diff in `help/`, but it's a neat trick nonetheless